### PR TITLE
114 scale correction

### PR DESCRIPTION
**Pull request by sgratiy**

### CODE REVIEW

File: pipeline/commands/scale_correction.py [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+    """
+
+    input_json_key = f"{run_prefix}/{reconstruction_id}.json"
+    input_json_response = s3.get_object(Bucket=working_bucket,
+                                        Key=input_json_key
+                                        )
+
+    input_data = json.load(input_json_response["Body"])
+
+    swc_key = f"{run_prefix}/{input_data['swc_file']}"
```
Comment (NileGraddis) [inline pipeline/commands/scale_correction.py:48]: Would be good for the landing step to track the raw swc key.

Comment (NileGraddis) [reply pipeline/commands/scale_correction.py:48]: not necessary for this pr

File: pipeline/commands/scale_correction.py [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+
+    return {
+        "morphology": morphology,
+        "soma_marker_z": soma_marker["z"],
+        "soma_depth": input_data["cell_depth"],
+        "cut_thickness": input_data["cut_thickness"],
+    }
+
+
+def morphology_to_s3(bucket: str, key: str, morphology: Morphology):
```
Comment (NileGraddis) [inline pipeline/commands/scale_correction.py:72]: we'll want to share this function among multiple steps

**Review by NileGraddis — APPROVED** [review]
LGTM! :christmas_tree:

**Review by gnayuy — APPROVED** [review]

**Pull request merged**